.circleci/config.yml

```diff
@@ -125,3 +125,59 @@ workflows:
           on_tag: false
           requires:
             - test-policies
+
+      - architect/push-to-app-collection:
```

Comment:
We do not want to push these to collections - they are included in `releases` and should therefor not be pushed to any collection :)

Reply:
Well, policies-shared should be as it's not part of a release or did I not understand how you want to deploy it ?

Reply:
This happens when I skim and don't read properly :D

Yes you are correct here :)

Just leave out the vmware collection for now I would say.

Reply:
As you wish Marcel the Genie Muller